javascript/main.js

```diff
@@ -0,0 +1,141 @@
+    drawerButton.classList.replace("bi-x", "bi-list");
+    drawerButton.style.position = "static";
+    buttonSwitch = false;
+  }
+});
+
+rollButton.addEventListener("click", () => {
+  devicePanel.style.display = "none";
+  drawerButton.classList.replace("bi-x", "bi-list");
+  drawerButton.style.position = "static";
+  buttonSwitch = false;
+});
+
+// Box Evangelist
+let arrContent = [];
+
+function objBuilder(name, awards, text, img_src) {
+  let newObj = new Object();
+  arrContent.push(newObj);
+
+  let i = arrContent.length - 1;
+
+  arrContent[i].name = name;
+  arrContent[i].awards = awards;
+  arrContent[i].text = text;
+  arrContent[i].img_src = img_src;
+}
+
+objBuilder(
+  "Dante Gebel",
+  "Preacher",
+  "He is currently pastor the megachurch River Church, in California, he is the author of several books written in Spanish.",
+  "./images/boxes/photo-dg.jpg"
+);
+objBuilder(
+  "Juan Carlos Harrigan",
+  "Evangelist",
+  "He is a renowned Evangelist who has been impacting people´s lives through his ministry for many years. He was born in Honduras and grew up in a family that was passionate about serving God.",
+  "./images/boxes/photo-jch.jpg"
+);
+objBuilder(
+  "José Duarte",
+  "Profet and Apostol",
+  "He studies on biblical Hebrew, eschatology, hermeneutics and everything that involves the Bible. Army Ministry of Revival of the Kingdom of God - Lambare Paraguay",
+  "./images/boxes/photo-jd.jpg"
+);
+objBuilder(
+  "Lamartine Posella",
+  "Preacher",
+  "Lamartine Posella is pastor president of YAH Church in São Paulo and works as a preacher, theologian, musician, composer, writer and lecturer.",
+  "./images/boxes/photo-lp.jpg"
+);
+
+// Box Evangelist
+let evangelistBoxesVessel = document.querySelector("#evangelist-speakers__boxes");
+let evangelistCoupledSpeakers = document.createElement("div");
+
+function htmlBuilder() {
+  for (let i = 0; i < arrContent.length; i++) {
+    if (i % 2 === 0) {
+      let newDiv = document.createElement("div");
+      newDiv.classList.add("m-tier", `evangelist-speakers__coupled-boxes${i}`);
+      evangelistBoxesVessel.insertAdjacentElement("beforeend", newDiv);
+      evangelistCoupledSpeakers = document.querySelector(
+        `.evangelist-speakers__coupled-boxes${i}`
+      );
+    }
+
+    evangelistCoupledSpeakers.insertAdjacentHTML(
+      "beforeend",
+      `
+        <div class="m-pil" id="evangelist-speakers__boxes">
+          <div class="evangelist-speakers__box">
+            <img class="evangelist-speakers__box-portrait" src="${arrContent[i].img_src}">
+            <div class="evangelist-speakers__box-texts m-pil">
+              <h3 class="evangelist-speakers__box-title">${arrContent[i].name}</h3>
+              <h4 class="evangelist-speakers__box-subtitle coral">${arrContent[i].awards}</h4>
+              <hr class="stripe-light" />
+              <p>${arrContent[i].text}</p>
+            </div>
+          </div>
+        </div>
+    `
+    );
+  }
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** To correctly match the template design on mobile version, I think it would be better if you improve the design of your speakers align them in `flex-direction: column` as right now here is how looks your design on mobile:
### Template design
![Screenshot from 2023-05-31 02-38-34](https://github.com/luisrpavanelli/capstone-project/assets/83042742/bddce990-e821-4bb9-bca3-06ca529ebeb4)

### Your design
![Screenshot from 2023-05-31 02-35-55](https://github.com/luisrpavanelli/capstone-project/assets/83042742/f2d608bb-9d03-4a2c-949d-eb39ee4f15a7)

**I can also suggest you two other more speakers to be 6 instead of 4 speakers**